src/routes/transactions.js

```diff
@@ -134,4 +134,37 @@ server.post("/transaction", isAuth, async (req, res) => {
   }
 });
 
+server.get('/transaction-history', isAuth, async (req, res) => {
+  const userId = req.session.passport.user._id;
+  const transactions = await Transaction.find({userId: userId});
+  const response = [];
+
+  if(transactions) {
```

Comment:
In order to get rid of if else statement you can return early here is no transaction is found
```suggestion
  if(!transactions) {
      retutrn res.status(204).json({
          message: "No transactions"
      });
  }
```